src/otx/algo/segmentation/dino_v2_seg.py

```diff
@@ -46,9 +46,8 @@ class OTXDinoV2Seg(TorchVisionCompatibleModel):
 
     def _create_model(self) -> nn.Module:
         # merge configurations with defaults overriding them
-        backbone_configuration = self.backbone_configuration | DinoV2Seg.default_backbone_configuration
-        decode_head_configuration = self.decode_head_configuration | DinoV2Seg.default_decode_head_configuration
-        # initialize backbones
+        backbone_configuration = DinoV2Seg.default_backbone_configuration | self.backbone_configuration
+        decode_head_configuration = DinoV2Seg.default_decode_head_configuration | self.decode_head_configuration
```

Comment:
Why is `self.**`_configuration separate, and is this something that can actually be used?

Reply:
a_dict | b_dict is same as a_dict.update(b_dict). So, self.**_configuration has a priority than default configuration.

Reply:
I added this default configuration for simple API usage. Let's imagine that we don't have it. Then user should create model and pass all parameters to initialize backbone and head. It is obvious overhead. Now, he can provide only few parameters to override default values.
self.** is what comes from config or what is passed to model class.